incubator/jaeger/templates/collector-deploy.yaml

```diff
@@ -45,8 +45,21 @@ spec:
 {{ toYaml .Values.collector.resources | indent 10 }}
         command:
         - "/go/bin/collector-linux"
+{{ if ne .Values.elasticsearch.config.host "" }}
```

Comment:
It's probably better to externalize this into a configmap, like we did recently on the jaegertracing/jaeger-kubernetes repository: 

https://github.com/jaegertracing/jaeger-kubernetes/blob/master/jaeger-production-template.yml#L40
https://github.com/jaegertracing/jaeger-kubernetes/blob/master/production/cassandra.yml#L18-L38

Reply:
That sounds like a good idea. I think it might be easier to have a second PR that cleans up all those settings into a ConfigMap. I don't want to confuse the two issues and make the PR more difficult to read.